docs/DevGuide.md

```diff
@@ -32,8 +32,6 @@ When running locally:
     - Check your environmental variables, you might have `AZURE_TENANT_ID`, `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` set, and the `DefaultAzureCredential` is attempting to use `EnvironmentalCredentials` for an app that doesn't have access to the dev KV.
```

Comment:
Maybe we could add the note here somewhere that if we're trying to flow something to or from the dotnet/dotnet repo for the first time, the cloning will take quite some time

Reply:
I am not sure it is necessary to make the dev guide longer. I think people sort of expect things to take some time when debugging locally and if they're curious what's going on, the logs will say quite clearly the service is cloning